release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -351,12 +351,12 @@ In the table below, features are marked with the following statuses:
 |Scheduler profiles
 |-
 |TP
-|
+|TP
```

Comment:
@damemi Can you confirm that scheduler profiles are staying tech preview for 4.8?

Reply:
@soltysh did these officially get moved to 4.9?

Reply:
Yes, GA in 4.9, as discussed in slack.